### ISSUE TITLE
One issue with device when transform_from_probs

### ISSUE THREAD

**Issue by julyanghar** [lucidrains/PaLM-rlhf-pytorch#67]
Hello! In hl_gauss.py:

`
    def transform_from_probs(self, probs):
        return (probs * self.centers).sum(dim = -1)
`
It seems that self.centers and probs might not be in the same device, which causes error.

I tried to alter this line in ppo.py and it works:

`
self.critic_hl_gauss_loss = HLGaussLoss(num_bins = critic_num_pred_bins, **hl_gauss_loss_kwargs).to(palm.device)
`
